.github/workflows/report_news.yaml

```diff
@@ -33,20 +35,23 @@ jobs:
         with:
           to: ${{ secrets.TELEGRAM_OSSCAMEROON_CHANNEL_ID }}
           token: ${{ secrets.TELEGRAM_BOT_TOKEN }}
-          message_file: ./.github/workflows/report_news/news_feed_message
+          message_file: |
+            ./.github/workflows/report_news/news_feed_message
 
-      - name: check the size of the hash-list and clean outdated hashes
+      - name: Check size of hash-list and clean outdated hashes
```

Comment:
Why removing the articles here ?

Reply:
😅  the string was too long for the linter, we can either rephrase or just put it in a new line with the `|` 🤷🏾‍♂️ ! 🤔